server/src/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -3556,6 +3556,11 @@ private void checkPublicIpRangeErrors(long zoneId, String vlanId, String vlanGat
             throw new InvalidParameterValueException("Please ensure that your end IP is in the same subnet as your IP range's gateway, as per the IP range's netmask.");
         }
         // check if the gatewayip is the part of the ip range being added.
+        // RFC 3021 - 31-Bit Prefixes on IPv4 Point-to-Point Links
```

Comment:
I talked to Remi yesterday and we agreed that would be nice to comment on the code so people don't remove the functionality by mistake or at least will make sure it still works if changes are made.

By fixing the bug CLOUDSTACK-4811 on October 2013, it made the /31 not work anymore.

The comment causes no harm.